dagster_polars/io_managers/utils.py

```diff
@@ -96,9 +114,8 @@ def get_metadata_table_and_schema(
             f"Polars sample:\n{df_sample}\n"
             f"dict sample:\n{pformat(df_sample.to_dicts())}"
         )
-        return schema, None
```

Comment:
I guess removing this is fine, I didn't see this branch being executed for a really long time now

Reply:
It's only refactored, not removed. In `get_polars_metadata` it will get the schema and then pass it into `get_metadata_and_table_schema`